CONTRIBUTING.md

```diff
@@ -135,9 +135,15 @@ Our puppet modules provide [`Gemfile`](./Gemfile)s which can tell a ruby
 package manager such as [bundler](https://bundler.io/) what Ruby packages,
 or Gems, are required to build, develop, and test this software.
 
-Please make sure you have [bundler installed](https://bundler.io/#getting-started)
-on your system, then use it to install all dependencies needed for this project,
-by running
+**Prerequisites**
+1. Please make sure you have [bundler installed](https://bundler.io/#getting-started)
+on your system.
+2. If you are using Fedora, you may need these additional packages
+    ```shell
+      % sudo dnf install -y ruby-devel redhat-rpm-config
```

Comment:
I typically install `rubygem-bundler` as well instead of installing bundler the regular way.

Reply:
yep, I will add this too!